www/activities/settings/views/statistics.html

```diff
@@ -72,6 +72,30 @@
             </div>
           </div>
         </li>
+
+        <li>
+          <div class="item-content">
+            <div class="item-inner">
+              <div class="item-title" data-localize="settings.statistics.send-data">Send data to own service</div>
```

Comment:
I think this could be worded better. Perhaps "Send data to external service"

Reply:
done